projects/plugins/jetpack/extensions/blocks/sharing-buttons/sharing-buttons.php

```diff
@@ -36,13 +36,8 @@ function render_block( $attr, $content ) {
 	if ( ! jetpack_is_frontend() ) {
 		return '';
 	}
-	return $content;
-}
 
-/**
- * Add the services list to the block
- */
-function add_sharing_buttons_block_data() {
+	// Add the services list to the block.
```

Comment:
Oh, actually, it seems that we could remove this part altogether. I think it was previously used for `view.js` script, but it doesn't rely on the services list anymore. What do you think?

Reply:
Happy to do that :) 5ffa0c3f3b4af7d2952968bedbdc63843700299e